src/bundle/Resources/public/scss/_typography.scss

```diff
@@ -37,3 +37,26 @@
         }
     }
 }
+
+.ez-adminsection-header {
+    border: none;
+
+    .card-header {
+        background-color: $ez-color-base-pale;
+
+        h2 {
```

Comment:
Can't we make it less specific? I mean you should create a class like: `.ez-card__title` and use it instead.
In the end you'll get: `.ez-card__title` instead of `.ez-adminsection-header .card-header h2`.
What's more, we should avoid styling using tag selectors. Only classes or atributes selectors.
Avoid nested selectors.